pkg/storage/s3/s3.go

```diff
@@ -419,7 +419,7 @@ func (d *driver) ConfigEnv() (envs envvar.List, err error) {
 		return
 	}
 
-	if currentFeatureGates.Enabled(util.ChunkSizeMiBFeatureGateName) && d.Config.ChunkSizeMiB > 0 {
+	if currentFeatureGates.Enabled(util.ChunkSizeMiBFeatureGateName) && d.Config.ChunkSizeMiB > 4 && d.Config.ChunkSizeMiB < 5121 {
```

Comment:
Is this range (4 and 5121) defined in the API validation as well? Just curious.

Reply:
Yes, minimum=5 and maximum 5120 in kubebuilder validation